src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -45,28 +46,6 @@ static VisualStudioInstanceFactory()
         public VisualStudioInstanceFactory()
         {
             AppDomain.CurrentDomain.AssemblyResolve += AssemblyResolveHandler;
-            AppDomain.CurrentDomain.FirstChanceException += FirstChanceExceptionHandler;
-        }
-
-        private static void FirstChanceExceptionHandler(object sender, FirstChanceExceptionEventArgs eventArgs)
```

Comment:
I see, too bad xunit requires this to get access to the logger.  I guess there isn't anything that can be done other than requiring test authors to add this in the constructor.